### ISSUE TITLE
Transforming IIFE results in extra parentheses

### ISSUE THREAD

**Issue by ghost** [lebab/lebab#187]
```
(function ($) {
  $.name = function (method) {};

  $(document).ready(function () {
    $('.someClass').name();
  });
})(jQuery);
```

give me 

```
(($ => {
  $.name = method => {};

  $(document).ready(() => {
    $('.someClass').name();
  });
}))(jQuery);
```

but 

```
($ => {
  $.name = method => {};

  $(document).ready(() => {
    $('.someClass').name();
  });
})(jQuery);
```

the same

**Comment by nene** [comment]
Aha, so the problem is that transforming:

    (function(){})()

produces:

    ((() => {}))()

which has one extra pair of parentheses.

Will take a look, but I suspect that the problem is in [Recast](https://github.com/benjamn/recast/), as Lebab itself only changes the abstract syntax tree, while Recast deals with printing it out with all the needed parenthesis.

**Comment by nene** [comment]
Opened an issue in Recast: https://github.com/benjamn/recast/issues/327

**Comment by nene** [comment]
Fixed in 3.0.2
